src/com/cft/contactmerge/contact/StreetMatchLogic.java

```diff
@@ -23,6 +23,9 @@
         put("cir", "circle");
         put("circle", "circle");
         put("blvd", "boulevard");
+        put("pw", "parkway");
```

Comment:
You'll need to double check, but I don't think that "pw" is a legitimate abbreviations for parkway. I don't think we should try to handle misspellings with these routines. At some point we should add some fuzzy logic comparison though (ones that aren't sensitive to minor misspellings).

Reply:
I did a little sleuthing, and we are missing some referenced from http://www.gis.co.clay.mn.us/usps.htm

Specifically,

PARKWAY == PKWY
PARKWAYS == PKWY

Then the obvious one PKWY == parkway.

Is this HashTable's purpose to check for alternative naming and then normalize the address? If so, this list could be an easy issue to bring someone on board to help with.

Reply:
I only added pw, because it was in the example you gave when we were looking at spreadsheets on Monday, but if you think it shouldn't be there, I will remove it.

Reply:
Awesome list Kevin! I will apply all those that are missing in the hash table.